tools/mnemonic_generator.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+
+# A simple/silly tool for generating a mnemonic with known words. Possibly useful for testing.
+# Usage:
+#  $ python tools/mnemonic_generator.py exotic toilet dance
+#  exotic toilet dance <rest of mnemonic>
+
+import argparse
+import sys
+
+from bitstring import BitArray
+from chia.util.keychain import bip39_word_list, bytes_to_mnemonic
+from secrets import token_bytes
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-l", "--list", action="store_true")
+    parser.add_argument("words", type=str, nargs="*", help="words to use in mnemonic (may specify up to 23)")
+
+    list = None
+    words = None
+
+    for key, value in vars(parser.parse_args()).items():
+        if key == "list":
+            list = value
+        elif key == "words":
+            words = value
+        else:
+            print(f"Invalid argument {key}")
+
+    if not list and len(words) == 0:
+        print("Example usage:")
+        print("python mnemonic_generator.py exotic toilet dance")
+        sys.exit(1)
+
+    word_list = bip39_word_list().splitlines()
+
+    if list:
+        for word in word_list:
+            print(word)
+        sys.exit(0)
+
+    bitarray = BitArray(token_bytes(32))
+
+    for i, word in enumerate(words):
+        try:
+            word_index = word_list.index(word)
+        except ValueError:
+            print(f"Word not found in BIP39 word list: {word}")
+            sys.exit(1)
+        start = i * 11
+        end = start + 11
+        bitarray[start:end] = word_index
```

Comment:
I don't recall the details of the bitstring library, but in general slice assignment can extend the size of the container.

```python-console
Python 3.8.10 (default, Jun 22 2022, 20:18:18) 
[GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> x = []
>>> x[3:5] = [1, 2]
>>> x
[1, 2]
```
This could happen here if we specified a full 24 words on the command line.